README.md

```diff
@@ -596,6 +596,7 @@ as possible. This is the list of fields:
 * `to: [<list>]`: a list of recipients of the email.
 * `cc: [<list>]`: a list of emails in cc.
 * `bcc: [<list>]`: a list of emails in bcc.
+* `replyto: [<string>]`: the email listed in the Reply-To field.
```

Comment:
Nit: please remove the square brackets as this is not a list (similarly to `subject: <string>`).

Reply:
Fixed!